client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
haha sorry this file re-appeared somehow ! you can remove it. there needs to be only a single `.gitignore` at the root.

Reply:
oopsy